src/PeopleLookup.Mvc/appsettings.json

```diff
@@ -5,10 +5,13 @@
     }
   },
   "Authentication": {
+    "Authority": "https://ssodev.ucdavis.edu/cas/oidc",
     "IamKey": "External",
     "CasBaseUrl": "https://ssodev.ucdavis.edu/cas/",
     "CasBaseUrlProd": "https://cas.ucdavis.edu/cas/",

```

Comment:
Remove these if they are not used anymore?

Reply:
yea i think we should remove the CAS* ones